integration/src/test/scala/ammonite/integration/BasicTests.scala

```diff
@@ -107,5 +108,20 @@ object BasicTests extends TestSuite{
       res.out.lines.foreach { println }
       println("\n-------------------------")
     }
+
+    'scriptCache{
+      val scripts = List('scriptLevelCaching/"testLoadModule.scala",
+        'scriptLevelCaching/"scriptTwo.scala",
+        'scriptLevelCaching/"scriptOne.scala",
+        'scriptLevelCaching/"QuickSort.scala")
```

Comment:
Format this (and [this](https://github.com/lihaoyi/Ammonite/pull/403/files#diff-d71dddf61413a0340636e6a9eaadee32R103) and [this](https://github.com/lihaoyi/Ammonite/pull/403/files#diff-bcdb6a0282e047eba770bc309743e114R400)) consistently with other multi-line functions:

```scala
FUNCTION(
  arg0,
  arg1,
  arg2,
)
```